lib/notion/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 module Notion
-  VERSION = '0.1.0-beta1'
-  NOTION_REQUEST_VERSION = '2021-08-16'
+  VERSION = '0.1.0-beta2'
```

Comment:
Let’s not bump the version number directly in this PR—once it’s merged, I’ll create a separate commit to create `1.0.0-beta1`. I believe your PR is the last one to get this gem to 1.0.0!